src/layouts/AccessWalletLayout/components/HardwareModal/HardwareModal.vue

```diff
@@ -130,14 +134,34 @@ export default {
       this.items.forEach(item => {
         const u2fhw = ['secalot', 'ledger', 'bitbox'];
         const inMobile = ['secalot', 'keepkey'];
+        const webUsb = ['keepkey'];
+
+        if (webUsb.includes(item.name)) {
+          const disable =
```

Comment:
Alternatively:

```suggestion
          const disable = !window || !window.navigator || !window.navigator.usb;
```

so it starts working if/when other browsers implement it. Not sure if there's a similar trick for detecting u2f support.

Reply:
Makes sense. I'm integrating this and adding the `https` check to it.